loader_hub/web/beautiful_soup_web/base.py

```diff
@@ -62,13 +66,22 @@ def _readmedocs_reader(soup: Any, url: str) -> Tuple[str, Dict[str, Any]]:
         page_link = requests.get(doc_link)
         soup = BeautifulSoup(page_link.text, "html.parser")
         try:
-            text = soup.find_all("article", {"id": "content"})[0].get_text()
+            text = ""
+            for element in soup.find_all("article", {"id": "content"}):
+                for child in element.descendants:
+                    if child.name == "a" and child.has_attr("href"):
```

Comment:
My one last request is - can we make this depend on a config parameter `include_url_in_text` in the __init__.py function that could be set to True by default? gives us extensability in case users want to include these links for other domain-specific parsers

Reply:
Okay, I'll look into to it.

Reply:
I've updated it.

Reply:
thanks. yeah i meant having it as an option in the `__init__` function (and then passing those params to these functions like _readmedocs_reader)